components/data-apps/index.md

```diff
@@ -0,0 +1,116 @@
+{% include warning.html content="In BETA we only support Github repositories." %}
+
+Please provide feedback using the feedback button in your project." %}
+If you have a complex application, you can push your app sources into Github and link the repository in this section.
+Provide Project URL, choose the right branch and finally select your main entrypoint file.
+
+{: .image-popup}
+![Git repository](/components/data-apps/data apps - git repository.png)
+
+## Secrets
+To provide your app with environment variables or sensitive information like credentials, api keys etc, enter them as key value pairs in the Secrets section.
+These secrets will be injected into the secrets.toml file upon deployment of the app. 
+[Read more about the Streamlit secrets](https://docs.streamlit.io/streamlit-community-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management).
+
+{: .image-popup}
+![Secrets](/components/data-apps/data apps - secrets.png)
+
+## Data loading
+To load Data into the app, you can either use [Input mapping](https://help.keboola.com/transformations/mappings/#input-mapping) section, which will load data from Keboola project Storage into the app.
+(Note that, while in BETA, the app needs to be redeployed to fetch up-to-date data.)
+Or you can use [Keboola Storage Python Client](https://github.com/keboola/sapi-python-client) in the app to load the data as needed.
+See [Examples](#Examples) below for usage of the Keboola Storage Python Client.
+
+## Deployment and App management
+
+{% include warning.html content="Once the Data App is deployed it's url will be publicly available! Keboola does not provide authorization to Data Apps out of the box yet." %}
+
+### Base Image
+When the app is deployed, the code specified in one of the deployment method will be injected into our [base Streamlit docker image](https://github.com/keboola/sandbox-streamlit/blob/main/Dockerfile). 
+This image already has Streamlit and few other basic packages pre-installed.
+
+### Actions menu
+
+- **Deploy Data app** - start the Data App, once the deployment job is finished, you can go to the Data App public url by clickin "Open Data App".
+- **Open Data App** - opens a new window with your Data App.
+- **Redeploy** - if you made changes in the Data App configuration, you have to redeploy it for the changes to take effect.
+- **Terminate Data App** - stop the Data App. The container in which the application is running will be stopped, and the apps url will no longer be available. The configuration of the app will remain intact.
+- **Delete Data App** - stop the Data App deployment and delete it's configuration.
+
+### Debugging App deployment
+If the Data App's deployment job fails, you can see the logs from its container in the events log of the deployment job.  
+For example a conflict of the specified packages:
+
+{: .image-popup}
+![Secrets](/components/data-apps/data apps - job error log.png)
+
+## Example Data Apps
+
+### Customer Segmentation
```

Comment:
Please add also this one: https://demo.keboola.com/app/data-apps/45663441
thanks.

Reply:
Jasne, sorry, ze som to tam nedal hned